src/resources/Catalogs/CatalogContent.ts

```diff
@@ -6,21 +6,11 @@ export type ObjectType = {
     objectType: string;
 };
 
-export type Version = {
-    version: number;
-};
-
 export default class CatalogContent extends Resource {
     static baseUrl = `/rest/organizations/${API.orgPlaceholder}/catalogcontent/source`;
 
-    getObjectTypeV2(sourceId: string, version: Version) {
-        return this.api.get<CatalogObjectType>(
-            this.buildPath(`${CatalogContent.baseUrl}/${sourceId}/objecttypes`, version),
-        );
-    }
-
-    getObjectTypes(sourceId: string) {
-        return this.api.get<string[]>(`${CatalogContent.baseUrl}/${sourceId}/objecttypes`);
+    getObjectTypeV2(sourceId: string) {
+        return this.api.get<CatalogObjectType>(this.buildPath(`${CatalogContent.baseUrl}/${sourceId}/objecttypes`));
```

Comment:
No need for the build path helper since there is no longer a query param to deal with.

```suggestion
    getObjectTypeV2(sourceId: string) {
        return this.api.get<CatalogObjectType>(`${CatalogContent.baseUrl}/${sourceId}/objecttypes`);
```